generate.go

```diff
@@ -98,9 +98,16 @@ func modify(spec *specs.LinuxSpec, rspec *specs.LinuxRuntimeSpec, context *cli.C
 	spec.Process.User.GID = uint32(context.Int("gid"))
 	rspec.Linux.SelinuxProcessLabel = context.String("selinux-label")
 
-	args := context.String("args")
-	if args != "" {
-		spec.Process.Args = []string{args}
+	for i, a := range context.StringSlice("args") {
+		if a == "" {
+			break
```

Comment:
@mrunalp     
If element in args is empty string, it is not an error here, the specs allow args to be empty value.
But it is not so clearly here to express the meaning in the code, therefore, I have modified it.
as below,
``` go
for i, a := range context.StringSlice("args") {
                if a != "" {
                        if i == 0 {
                                //Replace "sh" from getDefaultTemplate()
                                spec.Process.Args[0] = a
                        } else {
                                spec.Process.Args = append(spec.Process.Args, a)
                        }
                }
        }
```
Thanks for reminding.